src/styles/utils/_variables.scss

```
@@ -0,0 +1,6 @@
+$circle-size: 75vh;
+$center-size: 5%;
+
+$c-seconds-hand: #000;
+$c-minutes-hand: #000;
+$c-white: #000;
```

```suggestion
$c-black: #000;
```
Why do you need all these colors?